packages/loader/container-loader/src/test/connectionManager.spec.ts

```diff
@@ -216,12 +231,73 @@ describe("connectionManager", () => {
 			assert.strictEqual(connectionCount, 1, "Expect there to still not be a connection yet");
 			checkedTimeout = true;
 		}, 300);
-
 		connection = await waitForConnection();
 		assert.strictEqual(connectionCount, 2, "Expect there to be a connection after waiting");
 		assert(checkedTimeout, "Expected to have checked 300ms timeout");
 	});
 
+	it("Does not re-try connection on error if ReconnectMode=Disabled", async () => {
+		const connectionManager = createConnectionManager();
+		// mock connectToDeltaStream method so that it throws a retriable error when connect() is called in connectionManager
+		const stubbedConnectToDeltaStream = stub(mockDocumentService, "connectToDeltaStream");
```

Comment:
I do not see anywhere where we initialize mockDocumentService.

Reply:
Its declared in the beforeEach() callback.